generated/provider_dependencies.json

```diff
@@ -17,6 +17,7 @@
   },
   "amazon": {
     "deps": [
+      "aiobotocore>=2.1.1",
```

Comment:
Same comment as https://github.com/apache/airflow/pull/28850#discussion_r1066843322 applies here too

Reply:
Sure will look into that.